doc/classes/XRFaceTracker.xml

```diff
@@ -0,0 +1,469 @@
+		</constant>
+		<constant name="FT_LIP_SUCK_LOWER_RIGHT" value="48" enum="BlendShapeEntry">
+			Lower right lip part tucks in the mouth.
+		</constant>
+		<constant name="FT_LIP_SUCK_LOWER_LEFT" value="49" enum="BlendShapeEntry">
+			Lower left lip part tucks in the mouth.
+		</constant>
+		<constant name="FT_LIP_SUCK_CORNER_RIGHT" value="50" enum="BlendShapeEntry">
+			Right lip corner folds into the mouth.
+		</constant>
+		<constant name="FT_LIP_SUCK_CORNER_LEFT" value="51" enum="BlendShapeEntry">
+			Left lip corner folds into the mouth.
+		</constant>
+		<constant name="FT_LIP_FUNNEL_UPPER_RIGHT" value="52" enum="BlendShapeEntry">
+			Upper right lip part pushes into a funnel.
+		</constant>
+		<constant name="FT_LIP_FUNNEL_UPPER_LEFT" value="53" enum="BlendShapeEntry">
+			Upper left lip part pushes into a funnel.
+		</constant>
+		<constant name="FT_LIP_FUNNEL_LOWER_RIGHT" value="54" enum="BlendShapeEntry">
+			Lower right lip part pushes into a funnel.
+		</constant>
+		<constant name="FT_LIP_FUNNEL_LOWER_LEFT" value="55" enum="BlendShapeEntry">
+			Lower left lip part pushes into a funnel.
+		</constant>
+		<constant name="FT_LIP_PUCKER_UPPER_RIGHT" value="56" enum="BlendShapeEntry">
+			Upper right lip part pushes outwards.
+		</constant>
+		<constant name="FT_LIP_PUCKER_UPPER_LEFT" value="57" enum="BlendShapeEntry">
+			Upper left lip part pushes outwards.
+		</constant>
+		<constant name="FT_LIP_PUCKER_LOWER_RIGHT" value="58" enum="BlendShapeEntry">
+			Lower right lip part pushes outwards.
+		</constant>
+		<constant name="FT_LIP_PUCKER_LOWER_LEFT" value="59" enum="BlendShapeEntry">
+			Lower left lip part pushes outwards.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_UP_RIGHT" value="60" enum="BlendShapeEntry">
+			Upper right part of the lip pulls up.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_UP_LEFT" value="61" enum="BlendShapeEntry">
+			Upper left part of the lip pulls up.
+		</constant>
+		<constant name="FT_MOUTH_LOWER_DOWN_RIGHT" value="62" enum="BlendShapeEntry">
+			Lower right part of the lip pulls up.
+		</constant>
+		<constant name="FT_MOUTH_LOWER_DOWN_LEFT" value="63" enum="BlendShapeEntry">
+			Lower left part of the lip pulls up.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_DEEPEN_RIGHT" value="64" enum="BlendShapeEntry">
+			Upper right lip part pushes in the cheek.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_DEEPEN_LEFT" value="65" enum="BlendShapeEntry">
+			Upper left lip part pushes in the cheek.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_RIGHT" value="66" enum="BlendShapeEntry">
+			Moves upper lip right.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_LEFT" value="67" enum="BlendShapeEntry">
+			Moves upper lip left.
+		</constant>
+		<constant name="FT_MOUTH_LOWER_RIGHT" value="68" enum="BlendShapeEntry">
+			Moves lower lip right.
+		</constant>
+		<constant name="FT_MOUTH_LOWER_LEFT" value="69" enum="BlendShapeEntry">
+			Moves lower lip left.
+		</constant>
```

Comment:
These do not feel explanatory enough. Not a VR expert and I'm not sure what they are referring to, indeed.